### PR TITLE
Chinese translations

### PR DESCRIPTION
**Pull request by Smtbook**
I added Traditional and Simplfied Chinese translations to the theme, but I am unsure about the language codes for them.

### CODE REVIEW

**Comment by welpo** [comment]
Thank you so much, @Smtbook!

Let me think about the language codes and get back to you. Maybe we can use BCP 47.

File: i18n/zh-CN.toml [hunk 1]
```diff
@@ -27,36 +23,36 @@ all_tags = "所有标签"
...
-close_quotation_mark = "”"
+open_quotation_mark = "「"
+close_quotation_mark = "」"
 
 # Translations for stylised Atom feed.
 # https://welpo.github.io/tabi/atom.xml
 # Must contain "About Feeds"; it will become a link.
-about_feeds = "这是一个网页订阅源，也称为Atom订阅源。通过复制地址栏中的URL到您的新闻阅读器订阅。访问 About Feeds 了解更多并开始使用。免费。"
-visit_the_site = "访问网站"
+about_feeds = "这是Web Feed，又称为Atom Feed，把现在的网址复制到新闻阅读器即可订阅本站文章。造访「About Feed」来了解更多资讯。"
```
Comment (welpo) [inline i18n/zh-CN.toml:56]: should contain "About Feeds" (missing the s in "Feeds") so it becomes a link in the [Stylized feed](https://welpo.github.io/tabi/atom.xml)

File: i18n/zh-TW.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# https://welpo.github.io/tabi/blog/shortcodes/#multilingual-quotes
+show_original_quote = "顯示原文"
+show_translation = "顯示翻譯"
+open_quotation_mark = "「"
+close_quotation_mark = "」"
+
+# Translations for stylised Atom feed.
+# https://welpo.github.io/tabi/atom.xml
+# Must contain "About Feeds"; it will become a link.
+about_feeds = "這是Web Feed，又稱為Atom Feed，把現在的網址複製到新聞閱讀器即可訂閱本站文章。造訪「About Feed」來了解更多資訊。"
```
Comment (welpo) [inline i18n/zh-TW.toml:56]: should contain "About Feeds" (missing the s in "Feeds") so it becomes a link in the [Stylized feed](https://welpo.github.io/tabi/atom.xml)

**Review by welpo — CHANGES_REQUESTED** [review]
The `about_feeds` must contain "About Feeds" (in plural) so it becomes a link.

Regarding the filenames, I think it makes most sense to use [BCP-47 codes](https://en.wikipedia.org/wiki/IETF_language_tag).

So Simplified Chinese would be `zh-Hans`, and Traditional Chinese `zh-Hant`.

Thanks again!

**Comment by Smtbook** [comment]
The issues has been corrected now.

**Comment by Smtbook** [comment]
The issues has been corrected.

**Review by welpo — APPROVED** [review]

**Comment by welpo** [comment]
多谢!

**Pull request merged**